src/mod/applications/mod_signalwire/mod_signalwire.c

```diff
@@ -1084,18 +1151,28 @@ static void mod_signalwire_state_configure(void)
 
 	snprintf(external_endpoint, sizeof(external_endpoint), "%s:%u", external_ip, external_port);
 
+#if SIGNALWIRE_CLIENT_C_VERSION_MAJOR >= 2
+	if (!swclt_sess_provisioning_configure(globals.signalwire_session, "freeswitch", local_endpoint, external_endpoint, globals.relay_connector_id, &reply)) {
+		if (reply->type == SWCLT_CMD_TYPE_RESULT) {
+#else
 	if (!swclt_sess_provisioning_configure(globals.signalwire_session, "freeswitch", local_endpoint, external_endpoint, globals.relay_connector_id, &cmd)) {
 		SWCLT_CMD_TYPE cmd_type;
 		swclt_cmd_type(cmd, &cmd_type);
 		if (cmd_type == SWCLT_CMD_TYPE_RESULT) {
 			const ks_json_t *result;
+#endif
 			signalwire_provisioning_configure_response_t *configure_res;
-
-			swclt_cmd_result(cmd, &result);
-			result = ks_json_get_object_item(result, "result");
+#if SIGNALWIRE_CLIENT_C_VERSION_MAJOR >= 2
+			if (!SIGNALWIRE_PROVISIONING_CONFIGURE_RESPONSE_PARSE(reply->pool, reply->json, &configure_res)) {
+#else
 			if (!SIGNALWIRE_PROVISIONING_CONFIGURE_RESPONSE_PARSE(ks_handle_pool(cmd), result, &configure_res)) {
+#endif
```

Comment:
missing swclt_cmd_result... that was removed above

Reply:
Oh, good catch! Fixed.